### PR TITLE
https://issues.liferay.com/browse/LPS-77752

### PR DESCRIPTION
**Pull request by willnewbury**

### CODE REVIEW

**Comment by drewbrokke** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by willnewbury** [comment]
Forgot to switch the heading sizes - updated the pull

**Comment by willnewbury** [comment]
ci:test

File: modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/display/ManageUserAssociatedDataEntitiesDisplay.java [hunk 1]
```diff
@@ -39,7 +43,30 @@ public UADEntityDisplay getUADEntityDisplay() {
...
+
+	public void setCount(int count) {
+		_count = count;
+	}
+
+	public void setUAdEntitySetName(String uadEntitySetName) {
+		_uadEntitySetName = uadEntitySetName;
+	}
+
+	public void setUAdRegistryKey(String uadRegistryKey) {
```
Comment (drewbrokke) [inline modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/display/ManageUserAssociatedDataEntitiesDisplay.java]: These two setters should be named `setUAD*` (capitalize the 'D')

File: modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/display/ManageUserAssociatedDataEntitiesDisplay.java [hunk 2]
```diff
@@ -39,7 +43,30 @@ public UADEntityDisplay getUADEntityDisplay() {
...
+		_uadRegistryKey = uadRegistryKey;
+	}
+
+	private int _count;
 	private final UADEntityDisplay _uadEntityDisplay;
 	private final SearchContainer<UADEntity> _uadEntitySearchContainer;
```
Comment (drewbrokke) [inline modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/display/ManageUserAssociatedDataEntitiesDisplay.java]: Can you also create setters for these two fields and remove the constructor?

File: modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/display/ManageUserAssociatedDataEntitiesDisplay.java [hunk 3]
```diff
@@ -39,7 +43,30 @@ public UADEntityDisplay getUADEntityDisplay() {
...
+
+	public void setUAdEntitySetName(String uadEntitySetName) {
+		_uadEntitySetName = uadEntitySetName;
+	}
+
+	public void setUAdRegistryKey(String uadRegistryKey) {
+		_uadRegistryKey = uadRegistryKey;
+	}
+
+	private int _count;
```
Comment (drewbrokke) [inline modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/display/ManageUserAssociatedDataEntitiesDisplay.java]: The search container will already have the count in it, so we shouldn't need an extra field here.

File: modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/portlet/action/ManageUserAssociatedDataEntitiesMVCRenderCommand.java [hunk 4]
```diff
@@ -77,18 +77,33 @@ public String render(
...
+
+		ManageUserAssociatedDataEntitiesDisplay
+			manageUserAssociatedDataEntitiesDisplay =
+				new ManageUserAssociatedDataEntitiesDisplay(
+					_uadRegistry.getUADEntityDisplay(uadRegistryKey),
+					searchContainer);
+
+		manageUserAssociatedDataEntitiesDisplay.setCount(count);
+		manageUserAssociatedDataEntitiesDisplay.setUAdEntitySetName(
+			entitySetName);
```
Comment (drewbrokke) [inline modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/portlet/action/ManageUserAssociatedDataEntitiesMVCRenderCommand.java]: We can inline `entitySetName`.

File: modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/portlet/action/ManageUserAssociatedDataEntitiesMVCRenderCommand.java [hunk 5]
```diff
@@ -77,18 +77,33 @@ public String render(
...
+
+		searchContainer.setTotal(count);
+
+		String entitySetName = uadEntityAggregator.getUADEntitySetName();
+
+		ManageUserAssociatedDataEntitiesDisplay
+			manageUserAssociatedDataEntitiesDisplay =
+				new ManageUserAssociatedDataEntitiesDisplay(
+					_uadRegistry.getUADEntityDisplay(uadRegistryKey),
+					searchContainer);
```
Comment (drewbrokke) [inline modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/portlet/action/ManageUserAssociatedDataEntitiesMVCRenderCommand.java]: In a case like this, Brian will probably want us to create the Display class first (we can do that if we're using a nullary constructor), then call each setter alphabetically by method name, creating each piece of data before the respective setter if necessary.

**Pull request closed without merging**